fetch_github_repos.py

```
@@ -236,50 +242,19 @@ def clone_or_update_repos(repos, output_dir=None, use_ssh=True):
-                        name,
-                        "checkout",
-                        "-b",
-                        "main",
-                        "origin/main",
-                        "--no-track",
-                    ],
+                    ["git", "clone", url, name],
                     check=True,
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE,
                 )
```

Ruff (0.8.2)

Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)